src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/StaffRefDataServiceImpl.java

```diff
@@ -796,17 +783,13 @@ public CaseWorkerProfile processExistingCaseWorkers(
             if (isUserSuspended(UserProfileUpdatedData.builder().idamStatus(IDAM_STATUS_SUSPENDED).build(),
```

Comment:
suspending user at line 783 and again line no 961. It will made IDAM call twice to suspend user

Reply:
we are just setting the values to the object but that wont  call idam  ... as the namechange is false that will call only modify roles in UP

Reply:
agreed with @sahitya-desireddy  comments . marking it as resolved